code-challenges.js

```diff
@@ -12,53 +12,101 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+const divisible= num => {
 
+    if (num % 3 === 0 )  {
+     
+    
+    return `${num} is divisible by three.`;
+        
+    }
+        else {
+    
+    return `${num} is not divisible by three.`
+}
 
+}
+console.log(divisible(num1))
 
+// // --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
-// --------------------2) Create a function that takes in the randomNouns variable and returns an array with all the words capitalized.
 
 var randomNouns = ["streetlamp", "potato", "teeth", "conclusion", "nephew", "temperature", "database"]
-// Expected output: ["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew", "Temperature", "Database"]
 
+const capitalizer = (array) => {
+  let eachItem = array.map(value => {
+    return value[0].toUpperCase() + value.substring(1)
+  })
+  return eachItem.join(" ")
+}
+console.log(capitalizer(randomNouns))
 
+// // --------------------3) Create a function that takes in the mixedDataArray array and returns an array with ONLY NUMBERS sorted from least to greatest.
 
+var mixedDataArray = [true, 8, "hello", 90, -8, null, 0, 46, 59, 107, "hey!"];
+// // Expected output: [-8, 0, 8, 46, 59, 90, 107]
 
+const eachItem = mixedDataArray.filter(mixedDataArray = Number.isInteger)
```

Comment:
this line is a variable declaration, but should be wrapped into a function so that it can be reusable for any array we might want to pass it. also, it _does_ return only numbers, but misses the functionality of sorting the numbers from least to greatest.